tests/main/views/test_brief_response.py

```diff
@@ -550,32 +552,44 @@ def test_can_not_submit_a_brief_response_that_already_been_submitted(self, live_
         data = json.loads(repeat_res.get_data(as_text=True))
         assert data == {'error': 'Brief response must be a draft'}
 
-    def test_can_not_submit_a_brief_response_for_a_framework_that_is_not_live_or_expired(self, live_dos_framework):
-            with self.app.app_context():
-                for framework_status in [status for status in Framework.STATUSES if status not in ('live', 'expired')]:
```

Comment:
I think we set up this framework_statuses list comprehension so if we ever added a new framework status then we wouldn't need to manually add it to our list which I thought was kind of nice... It's not a game changer though...